Tasks/AzureAppServiceManage/operations/KuduServiceUtils.ts

```diff
@@ -45,17 +45,29 @@ export class KuduServiceUtils {
         outputVariables = outputVariables ? outputVariables : [];
         var outputVariableIterator: number = 0;
         var siteExtensions = await this._appServiceKuduService.getSiteExtensions();
+        var allSiteExtensions = await this._appServiceKuduService.getAllSiteExtensions();
         var anyExtensionInstalled: boolean = false;
         var siteExtensionMap = {};
+        var allSiteExtensionMap = {};
         var extensionLocalPaths: string = "";
         for(var siteExtension of siteExtensions) {
             siteExtensionMap[siteExtension.id] = siteExtension;
         }
+        for(var siteExtension of allSiteExtensions) {
+            allSiteExtensionMap[siteExtension.id] = siteExtension;
```

Comment:
suggestion: Can we include the title too in the map?

Reply:
added